client/state/timezones/reducer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import { TIMEZONES_RECEIVE } from 'state/action-types';
+
+import {
+	rawOffsetsSchema,
+	labelsSchema,
+	continentsSchema
+} from './schema';
+
+export const rawOffsetReducer = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, { rawOffsets = [] } ) => rawOffsets
```

Comment:
this is a set-and-replace. there might be utility in folding in the new results with the old ones.